pkgs/top-level/all-packages.nix

```diff
@@ -1047,7 +1047,10 @@ with pkgs;
 
   devmem2 = callPackage ../os-specific/linux/devmem2 { };
 
-  dbus-broker = callPackage ../os-specific/linux/dbus-broker {};
+  # dbus-broker needs kernel 4.14+
+  dbus-broker = callPackage ../os-specific/linux/dbus-broker {
+    linuxHeaders = linuxHeaders_4_15;
```

Comment:
Just delete this line. There is only `linuxHeaders` by now, which is equal to `linuxHeaders_4_15`.